### PR TITLE
37 feature create account page

### PR DESCRIPTION
**Pull request by akajugz**
Created the elements in the account page and updated the readme file

### CODE REVIEW

File: src/pages/Account/Account.scss [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    padding: 0 0 1rem 0;
+    &--box {
+      font-size: 0.8rem;
+      line-height: 20px;
+      width: 100%;
+      padding: 0;
+      &:active,
+      &:focus {
+        outline: none;
+      }
```
Comment (olirob93) [inline src/pages/Account/Account.scss:72]: take a look at the padding here, it looks a bit strange the input is miles away from the underline. Not sure if it's this scss or somewhere else

**Review by olirob93 — CHANGES_REQUESTED** [review]
small change then all good

**Review by olirob93 — APPROVED** [review]
looks good to me

File: src/pages/Account/Account.tsx [hunk 2]
```diff
@@ -1,18 +1,47 @@
...
+  }
 
   return (
     <Layout>
-      <Header
-        title={"Welcome Back"}
-        subTitle="Name goes here..."
-        
-        />
+      <Header title={"Welcome Back"} />
```
Comment (Charlie-robin) [inline src/pages/Account/Account.tsx:19]: Let us change this to something along the lines of "Manage your Account"

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
One small change please

**Pull request merged**